src/content/basics/kubernetes-on-giant-swarm/index.md

```diff
@@ -1,10 +1,11 @@
 +++
 title = "Details on Kubernetes on Giant Swarm"
 description = "Here you learn how we set up things for you and what we manage, so you don't have to."
-date = "2019-11-14"
+date = "2019-11-18"
 weight = 10
 type = "page"
 categories = ["basics"]
+last review date = "2019-11-18"
```

Comment:
```suggestion
last_review_date = "2019-11-18"
```

Our site generator does not like to have spaces in front matter attribute names.